apps/accounts/forms/update.py

```diff
@@ -12,6 +12,7 @@
 
 # HTK Imports
 from htk.apps.accounts.emails import password_changed_email
+from htk.constants.i18n.timezones import US_TIMEZONE_CHOICES
```

Comment:
remove, this should be passed in from the call site that initializes this form

Reply:
Done